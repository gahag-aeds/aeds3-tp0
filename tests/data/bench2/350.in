350 175
63 76 35 21 95 8 18 88 27 60 61 38 73 78 72 17 99 36 58 62 0 82 24 2 79 43 12 55 57 83 20 73 59 7 94 6 15 12 95 94 25 8 85 50 38 57 67 37 93 77 51 46 11 75 48 90 70 12 45 28 47 17 1 59 76 47 65 91 60 12 38 37 20 23 87 10 80 7 99 26 36 2 72 48 78 72 38 0 36 36 80 84 5 33 95 82 81 60 25 93 73 63 30 45 86 17 8 19 76 7 45 13 10 69 61 40 41 51 40 77 87 73 13 45 6 8 27 87 21 52 80 46 68 62 43 54 32 51 25 8 59 70 73 21 39 34 61 32 38 53 10 77 26 75 22 85 36 1 72 57 6 5 55 74 67 98 80 99 50 6 60 61 76 33 82 68 20 95 0 58 0 62 35 79 90 10 64 26 63 88 35 69 93 90 95 61 88 76 12 38 82 72 51 10 6 85 30 78 32 83 36 33 45 23 64 35 85 28 61 49 16 96 18 62 86 14 75 27 42 87 17 76 60 69 38 18 6 69 96 39 52 84 24 49 59 88 37 45 68 98 94 36 47 64 98 85 30 73 64 24 61 82 0 73 3 39 43 9 60 91 0 64 75 24 13 34 64 50 79 84 1 25 21 0 42 19 37 72 93 2 49 6 84 49 79 39 88 74 48 0 65 49 64 40 25 78 26 90 80 58 74 81 83 47 33 25 67 71 50 12 73 99 18 9 0 49 48 41 23 48 41 88 49 6 80 75 36 6 17 68 64 43 50 0 91 35 77 58 6 79 
Add 83 198
Min 112 227
Add 159 274
Min 169 284
Add 163 278
Min 130 245
Sub 96 211
Sum 75 190
Add 123 238
Max 215 330
Sub 16 131
Max 62 177
Add 207 322
Min 91 206
Sub 124 239
Sum 201 316
Sub 74 189
Max 152 267
Add 19 134
Sum 177 292
Add 73 188
Min 220 335
Add 187 302
Sum 159 274
Sub 154 269
Max 188 303
Sub 200 315
Sum 189 304
Sub 40 155
Max 17 132
Add 97 212
Max 122 237
Sub 234 349
Min 127 242
Sub 162 277
Min 163 278
Sub 128 243
Max 23 138
Add 3 118
Min 16 131
Add 109 224
Min 124 239
Sub 184 299
Sum 81 196
Add 87 202
Max 179 294
Add 153 268
Min 160 275
Add 97 212
Min 43 158
Add 209 324
Max 41 156
Sub 28 143
Sum 33 148
Add 199 314
Max 53 168
Sub 220 335
Sum 36 151
Sub 114 229
Max 131 246
Sub 158 273
Min 82 197
Sub 124 239
Max 158 273
Sub 80 195
Sum 51 166
Sub 192 307
Min 208 323
Add 179 294
Sum 66 181
Add 223 338
Max 53 168
Sub 62 177
Max 44 159
Add 5 120
Min 148 263
Add 223 338
Max 29 144
Add 179 294
Max 191 306
Add 177 292
Min 25 140
Add 231 346
Max 77 192
Add 57 172
Min 67 182
Add 129 244
Min 148 263
Sub 208 323
Sum 114 229
Sub 150 265
Sum 132 247
Add 195 310
Sum 39 154
Add 161 276
Max 41 156
Add 195 310
Min 118 233
Sub 120 235
Min 139 254
Add 55 170
Sum 108 223
Add 63 178
Max 116 231
Sub 24 139
Max 173 288
Add 29 144
Sum 12 127
Add 73 188
Max 80 195
Sub 74 189
Min 16 131
Sub 104 219
Max 176 291
Sub 92 207
Sum 33 148
Sub 8 123
Max 92 207
Sub 180 295
Max 215 330
Sub 78 193
Sum 96 211
Sub 218 333
Max 38 153
Sub 6 121
Sum 144 259
Sub 184 299
Sum 201 316
Add 27 142
Sum 69 184
Add 211 326
Min 82 197
Add 49 164
Max 146 261
Add 69 184
Sum 72 187
Sub 190 305
Max 98 213
Add 189 304
Max 29 144
Add 49 164
Min 28 143
Sub 44 159
Sum 24 139
Add 203 318
Min 7 122
Sub 56 171
Min 82 197
Sub 204 319
Max 2 117
Sub 168 283
Max 47 162
Add 203 318
Max 152 267
Add 85 200
Min 208 323
Add 61 176
Max 140 255
Sub 46 161
Max 194 309
Add 81 196
Sum 129 244
Add 147 262
Sum 129 244
Sub 146 261
Max 215 330
Sub 72 187
Max 101 216
Sub 184 299
Max 26 141
Add 129 244
Max 104 219
Add 53 168
Max 44 159
Sub 128 243
