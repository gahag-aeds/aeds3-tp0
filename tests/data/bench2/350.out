1
0
2
5748
100
102
2
6162
101
6139
0
6607
101
6207
98
101
1
0
98
1
1
5898
97
0
1
99
5570
103
5517
102
1
100
5382
-1
5512
99
97
-1
98
101
0
98
0
1
6057
6110
5630
99
0
-1
5931
101
104
5772
102
2
104
5609
99
103
5748
99
6253
6139
5428
-2
102
5677
99
101
4
5662
0
-1
100
100
102
4
102
102
6076
6174
102
99
99
100
100
